backend/core/src/seeder/seeds/jurisdictions.ts

```diff
@@ -9,7 +9,7 @@ export const activeJurisdictions: JurisdictionCreateDto[] = [
     multiselectQuestions: [],
     languages: [Language.en],
     publicUrl: "",
-    notificationsSignUpURL: "https://public.govdelivery.com/accounts/CAALAME/signup/29652",
+    notificationsSignUpURL: "https://public.govdelivery.com/accounts/CAMTC/signup/36832",
```

Comment:
(related to the magic strings vs. env variables suggestion in an earlier PR) we need to change this in `testHelpers.ts` as well